src/lib/fields_derivers_graphql/fields_derivers_graphql.ml

```diff
@@ -158,6 +158,19 @@ module Graphql_fields_raw = struct
       (obj#nullable_graphql_fields := Input.T.{ run = (fun () -> Schema.bool) }) ;
       obj
 
+    let unit obj =
```

Comment:
there are quite a few `unit`, for example on feePayer where it differs from the normal party. I think null is a fundamental type that we need :D

Reply:
From the perspective of someone consuming the GraphQL API, I disagree I think we shouldn't include that field at all if it's always null.

You could argue that this will only be used by machines in which case, it may be okay (but the obj.magic used below probably may cause a crash at runtime)

Reply:
I thought someone better at OCaml might be able to remove the Obj.magic. It's something about these `for all x: f(x)` types Would you take a look?